website/docs/installation/documentdb/kubernetes.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 3
+---
+
+# Kubernetes
+
+FerretDB uses PostgreSQL with [DocumentDB extension](https://github.com/microsoft/documentdb) as a database engine.
+You need to have a running Kubernetes cluster to set up PostgreSQL with DocumentDB extension on Kubernetes.
+
+You can deploy PostgreSQL with DocumentDB extension using any of our provided images.
+Please see the [Docker installation docs](../documentdb/docker.md) to learn more about the available images.
+
+:::tip
+We strongly recommend specifying the full image tag (e.g., `17-0.102.0-ferretdb-2.1.0`)
```

Comment:
Could you also use an existing tag here please

Reply:
PTAL